src/simulator/spec/testData/gates-testdata.json

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
we'll need to check what all constants/configs we have and move them to /constants and import from there.

Reply:
but we can discuss this later.

Reply:
Okay